Условные операторы 3/Условные операторы 3/Program.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+
+namespace JapaneseCalendar
+{
+    class Program
+    {
+        static void Main()
+        {
+            while (true)
+            {
+                string str;
+
+                Console.WriteLine("Input the year number (press Enter to exit)");
+                str = Console.ReadLine();
+                if (str == string.Empty)
+                    break;
+                
+                int year;
+
+                if (!int.TryParse(str, out year))
+                {
+                    Console.WriteLine("Input error");
+                    continue;
+                }
+
+                int numberOfYear;
+
+                if (year >= 1984)
+                    numberOfYear = (year - 1984) % 60;
+                else
+                    numberOfYear = (year - 1984) % 60 + 60;
+
+                Console.WriteLine("{0} is the year of {1} {2}.",
```

Comment:
Хорошо бы исправить (мою) ошибку.
```suggestion
                Console.WriteLine("{0} is the year of a {1} {2}.",
```